src/main/java/javax/jmdns/impl/ServiceInfoImpl.java

```diff
@@ -451,6 +451,14 @@ public String getServer() {
         return (_server != null ? _server : "");
```

Comment:
Isn't this rather the bug here? It looks weird to return an empty string if no server is set... Did you manage to find out, why this is done this way?

Reply:
Perhaps it is, but given that it has been that was for 9 years I was hesitant to change it without fully understanding why it is there. I know that builds newer than that line of code have not had the particular bug I was trying to correct.